config.go

```diff
@@ -604,7 +606,10 @@ func (s *Settings) CreateInstallationToken(ctx context.Context, owner, repo stri
 			"appId":   authFields.AppId,
 			"ticket":  "EVG-19966",
 		}))
-		return "", errors.Wrap(err, "finding installation id")
+		if strings.Contains(err.Error(), "404") {
```

Comment:
Do we need this error at all? Seems like we could just not return an error here instead, and then we'll hit line 633 normally

Reply:
I am also wondering why this isn't an error though. Don't we need to know if the github app isn't installed?

Reply:
discussed to keep it since its useful to know when the app is not installed vs there was a problem getting the app